dvc/tree/base.py

```diff
@@ -294,8 +295,8 @@ def path_to_hash(self, path):
         return "".join(parts)
 
     def save_info(self, path_info, **kwargs):
-        typ, hash_ = self.get_hash(path_info, **kwargs)
-        return {typ: hash_}
+        hash_info = self.get_hash(path_info, **kwargs)
+        return {hash_info.name: hash_info.value}
```

Comment:
Let's create `hash_info.to_dict()` here?

Reply:
@skshetry `save_info` will go away soon, so I've decided not to implement it for now. `to_dict` might be useful in serialization (as well as `from_dict` in loading), but again no need to add it in this initial PR, especially since loading/dumping will be revisited in general.